src/main/resources/reference.conf

```diff
@@ -24,6 +24,20 @@ akka {
         # How often to sweep out expired cache entries.
         # Note that this interval has nothing to do with TTLs
         cache-cleanup-interval = 120s
+
+        # The IPs of nameservers to use e.g.
+        # akka.io.dns.async-dns {
+        #   nameservers = ["8.8.8.8", "8.8.4.4"]
+        # }
+        nameservers = []
+
+        # Whether to use the system's /etc/resolv.conf in place
+        # of nameservers
+        resolv-conf = off
```

Comment:
If resolve-conf = on overrides the use of nameservers, then I think logging an info that resolve-conf is being used instead of "nameservers = […]" might be a good thing so people don't expect both to be used?